website/docs/r/is_bare_metal_server.markdown

```diff
@@ -161,6 +161,12 @@ Review the argument references that you can specify for your resource.
       - `reserved_ip` - (Required, String) The unique identifier for this reserved IP.
       - `name` - (Required, String) The name for this reserved IP. The name is unique across all reserved IPs in a subnet.
       - `resource_type` - (Computed, String) The resource type.
+    - `protocol_state_filtering_mode` - (Optional, String) The protocol state filtering mode to use for this virtual network interface. 
```

Comment:
missing data source documentation

Reply:
only the VNI and instance template API response contains protocol_state_filtering_mode. So only the data sources for VNI and VNI collection and template has the changes.
Test cases are added